pyanaconda/anaconda_argparse.py

```diff
@@ -383,13 +385,20 @@ def getArgumentParser(version_string, boot_cmdline=None):
     # Version
     ap.add_argument('--version', action='version', version="%(prog)s " + version_string)
 
+    class ParseCmdlineMode(Action):
```

Comment:
I would suggest `SetCmdlineMode` (or something similar) as the Action subclass actually doesn't do any parsing in this case.